server/src/daemon/service.rs

```diff
@@ -66,6 +66,13 @@ impl Service {
                 .unwrap(),
             Arc::clone(&self.keyring),
         );
+        self.set_collections(Collection::new(
+            collection.label(),
+            alias,
+            Duration::from_secs(collection.created()),
```

Comment:
I would rename the current created/modified methods to return a Duration type and have variants that are exposed as dbus properties that are called `created_as_secs` / `modified_as_secs`

Reply:
done, thanks!